http-clients/src/main/java/org/triplea/http/client/github/issues/CreateIssueRequest.java

```diff
@@ -0,0 +1,28 @@
+package org.triplea.http.client.github.issues;
+
+import com.google.common.base.Ascii;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+/** Represents request data to create a github issue. */
+@ToString
+@EqualsAndHashCode
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class CreateIssueRequest {
+  private String title;
+  private String body;
+  private String[] labels;
+
+  public String getTitle() {
+    return title == null ? null : Ascii.truncate(title, 125, "...");
+  }
+
+  public String getBody() {
+    return body == null ? null : Ascii.truncate(body, 20000, "...");
```

Comment:
Is 20000 a github limit?

Reply:
no, it's somewhat arbitrary. This request object is lifted from the error upload request object. Before we had the same request object between client, server, and github. So this limit is just carried forward from that previous object. The fact that they match now I'm not sure is actually important. It still might be better to document that this is an arbitrarily large number.